common/source/docs/common-triducer.rst

```diff
@@ -0,0 +1,40 @@
+.. _common-triducer:
+
+[copywiki destination="copter,plane,rover"]
+
+=======================
+Triducer
+=======================
```

Comment:
too long ! it should match the text, same for the upper one's

Reply:
ok, thanks